src/shared/bus-unit-util.c

```diff
@@ -747,40 +803,6 @@ static int bus_append_cgroup_property(sd_bus_message *m, const char *field, cons
                 return 1;
         }
 
-        if (cgroup_io_limit_type_from_string(field) >= 0) {
```

Comment:
why is this block moving? the commit message doesn't say

Reply:
> Various functions are adjusted in a few places so that properties which
are support through an additional indirect function are processed last.
This makes the lists clearer.